vic/vic_run/src/snow_utility.c

```diff
@@ -111,8 +111,11 @@ snow_density(snow_data_struct *snow,
             c3 = exp(-0.046 * (density - dm));
             c4 = 1.0;
         }
-        if ((snow->surf_water + snow->pack_water) / snow->depth > 0.01) {
-            c4 = 2.0; /* presence of wet snow */
+
+        if (snow->depth > 0) {
+            if ((snow->surf_water + snow->pack_water) / snow->depth > 0.01) {
```

Comment:
What are the hard-coded `0.01` and `2.0`? Can we pull those out to one of the header files? If not here, then open an issue to collect hard-coded literals and refer to these.